conformance/test/_util.py

```diff
@@ -1,7 +1,7 @@
 import asyncio
 import sys
 
-VERSION_CONFORMANCE = "v1.0.4"
+VERSION_CONFORMANCE = "0bed9ca203aeda4e344edc442a4b2ede91726db5"
```

Comment:
want to create an issue over in https://github.com/connectrpc/conformance/issues to tag a new version for https://github.com/connectrpc/conformance/pull/1047? cc @jhump @doriable